internal/repos/syncer.go

```diff
@@ -589,6 +589,11 @@ func (s *Syncer) SyncExternalService(
 	seen := make(map[api.RepoID]struct{})
 	var errs error
 	fatal := func(err error) bool {
+		// If the error is just a warning, then it is not fatal.
+		var ref errors.Warning
+		if err != nil && errors.As(err, &ref) {
+			return false
+		}
 		return errcode.IsUnauthorized(err) ||
```

Comment:
Nit: new line here should improve readability. 😀 
```suggestion

		return errcode.IsUnauthorized(err) ||
```